include/grpc++/security/auth_context.h

```diff
@@ -37,6 +37,7 @@
 #include <iterator>
 #include <vector>
 
+#include <grpc/grpc_security.h>
```

Comment:
Ok, after offline chat, undoing these changes in this PR: we are mixing the secure and unsecure worlds.